src/main/java/org/example/globalgoodsindex/controllers/BarChartController.java

```diff
@@ -122,10 +155,10 @@ private Product findProductByName(String name) {
 
     private Tooltip createTooltip(String countryName, double salary, double price, int productsCount) {
         return new Tooltip(
-                "Country: " + countryName + "\n" +
-                        "Average Salary: $" + salary + "\n" +
-                        "Product Price: $" + price + "\n" +
-                        "Product Purchasable: " + productsCount
+                L10N.get("country") + ": " + countryName + "\n" +
+                        L10N.get("average_salary") + ": " + salary + "\n" +
+                        L10N.get("product_price") + ": " + price + "\n" +
+                        L10N.get("product_purchasable") + ": " + productsCount
         );
```

Comment:
maybe we should add here and the `productName`? to be more clear

Reply:
Maybe...

Reply:
![image](https://github.com/user-attachments/assets/e339271b-7d85-4012-b8bf-157d9e46254a)
![image](https://github.com/user-attachments/assets/56973301-33e6-4835-b068-06e9831ae093)
![image](https://github.com/user-attachments/assets/f24f5f12-a439-4306-930f-c497281e441f)

Reply:
We can also do it like this:
![image](https://github.com/user-attachments/assets/13554f20-c9b3-4a6f-8096-672e8814b701)